composer/src/lib.rs

```diff
@@ -55,7 +55,7 @@ impl RpcHandle {
         name: String,
         endpoint: SocketAddr,
     ) -> Result<Self, String> {
-        let mut attempts = 60;
+        let mut attempts = 40;
         loop {
```

Comment:
Is this the sort of thing that ought to be tunable via a configuration file or some such?

Reply:
Well, this is just the test library, I had to add a bigger delay before because of the nats lookup but managed to reduce it now by specifying a port. I can make it a default value part of Self which could be overridden.